lib/usd/translators/meshWriter_BlendShapes.cpp

```diff
@@ -505,19 +527,31 @@ void findUnionAndProcessArrays(
     for (size_t i = 0; i < numArrays; ++i) {
         const VtVec3fArray& origOffsetsArray = offsetsArrays[i];
         const size_t        numOrigOffsets = origOffsetsArray.size();
-        TF_VERIFY(numOrigOffsets != 0);
-        VtVec3fArray& newOffsetsArray = unionOffsetsArrays[i];
+        VtVec3fArray&       newOffsetsArray = unionOffsetsArrays[i];
         newOffsetsArray.assign(numUnionIndices, GfVec3f(0.0f));
 
         const VtVec3fArray& origNormalsArray = normalsArrays[i];
+        const size_t        numOrigNormals = origNormalsArray.size();
 #ifdef _DEBUG
```

Comment:
I think we don't need the _DEBUG preprocessor macro anymore.

Reply:
Resolved in 522b6b0.